### PR TITLE
New branch

### PR DESCRIPTION
**Pull request by ansuman-shukla**
## Purpose / Description
_Wiki text was not self explanatory_
## Fixes
* Fixes #

## Approach
_Updated the text to make it self explanatory_
## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration (SDK version(s), emulator or physical, etc)

## Learning (optional, can help others)
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

## Checklist
_Please, go through these checks before submitting the PR._

- [ ] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ ] You have commented your code, particularly in hard-to-understand areas
- [ ] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ Join Us On
 <a href="https://discord.gg/qjzcRTx"><img src="docs/graphics/logos/discord_logo_color.png" height="50px"/></a>
 <a href="https://www.reddit.com/r/Anki"><img src="docs/graphics/logos/reddit_logo_color.png" height="50px"/></a>
 <a href="https://www.facebook.com/AnkiDroid/"><img src="docs/graphics/logos/facebook_logo_color.png" height="50px"/></a>
-<a href="https://twitter.com/ankidroid"><img src="docs/graphics/logos/twitter_logo_color.png" height="50px"/></a>
+<a href="https://x.com/ankidroid"><img src="docs/graphics/logos/twitter_logo_color.png" height="50px"/></a>
```
Comment (Arthur-Milchior) [inline README.md]: Do you think I could also ask you to change the logo maybe, please ?

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (Arthur-Milchior) [inline .vscode/settings.json:1]: Why this change?
Maybe it's a good idea, but at least, please, explain what it does in the commit message. And maybe it could be its own commit. Right now a quick online search does not make it cleaar  we want it

**Review by Arthur-Milchior — COMMENTED** [review]
First commit, why not. Second one, I'd love at least a screenshot of before and after. 
Also, if we go full html, I think we should change the extension to html

But honestly, I don't see the point of moving to html.
It's not even like we expect users to see this page. This is mainly for dev. 
At least "Read the wiki" sounds a little bit better, I'll admit that. 


Finally, please, don't call it "new branch". Explain what you're doing in this PR so it's easy for reviewers to decide whether it's a part of the code they want to deal with. And also make a clear pull request message. Remove anything you don't fill, and explain what you do. Or I guess, in this case, why you want to do it

File: README.md (hunk 1 above)
Comment (ansuman-shukla) [reply README.md]: Yeah sure

**Comment by ansuman-shukla** [comment]
hey @Arthur-Milchior  ,
I'm getting started in this open-source community by making small contributions to know how this all works, thanks for your guidance next time I'll keep these things in mind

**Comment by david-allison** [comment]
Hi! I don't think we should convert our README to HTML, especially without a reason/discussion

If there's a reason, let's talk about it, but otherwise I'll propose closing this

PS: Thanks for the Twitter -> X fix, that can definitely be merged in your other PR

**Comment by BrayanDSO** [comment]
I agree with David. Closing this.
